Applications/DataExplorer/VtkVis/VtkCompositeTextureOnSurfaceFilter.cpp

```diff
@@ -61,9 +63,14 @@ void VtkCompositeTextureOnSurfaceFilter::init()
 
     QWidget* parent = nullptr;
     QSettings settings;
-    QString fileName = QFileDialog::getOpenFileName(parent, "Select raster file to apply as texture",
-                                                    settings.value("lastOpenedTextureFileDirectory").toString(),
-                                                    "Raster files (*.asc *.grd *.bmp *.jpg *.png *.tif);;NetCDF files (*.nc);;");
+    QString fileName = QFileDialog::getOpenFileName(
+        parent, "Select raster file to apply as texture",
+        settings.value("lastOpenedTextureFileDirectory").toString(),
+        "Raster files (*.asc *.grd *.bmp *.jpg *.png *.tif);;"
+#ifdef OGS_USE_NETCDF
+        "NetCDF files "
+#endif  // OGS_USE_NETCDF
```

Comment:
Should the `#endif` moved one line below?

Reply:
Thanks!